app/src/test/java/org/traccar/client/ProtocolFormatterTest.java

```diff
@@ -22,10 +22,17 @@ public void testFormatRequest() throws Exception {
         Position position = new Position("123456789012345", new Location("gps"), 0);
         position.setTime(new Date(0));
 
-        String url = ProtocolFormatter.formatRequest("localhost", 5055, false, position);
+        String url = ProtocolFormatter.formatRequest("http://localhost:5055/path", position);
```

Comment:
There should be at least one test without path because that's what Traccar Client will use by default.